### DISCUSSION TITLE
Ruby language support

### DISCUSSION THREAD

**Discussion by kingsleyh** [autometrics-dev/autometrics-shared#89] [General]
Would it be hard to add Ruby support? I see there is python support - maybe it would be similar to that?

**Comment by brettimus** [comment]
hey! i started work for this back in february, but after visiting some meetups and talking to a few teams, i didn't see much demand and stopped developing it.

also a big blocker for me was how difficult the dev tooling was proving to be (getting tooltips in vscode)

the repo is here if you wanna take a peak: https://github.com/autometrics-dev/autometrics-rb

then, i tried writing a solargraph plugin (https://github.com/autometrics-dev/solargraph-autometrics) for the tooltips, but i hit an issue with that language server's plugins

https://github.com/castwide/solargraph/issues/636

**Comment by brettimus** [comment]
are you using prometheus in a ruby app?
